modules/Library.js

```diff
@@ -0,0 +1,47 @@
+// import { bookSection } from '../index.js';
```

Comment:
- [ ] Kindly remove all unnecessary codes from the file. Doing so makes your code clean. 👍